amethyst_renderer/src/vertex.rs

```diff
@@ -185,11 +187,11 @@ impl With<Color> for PosColor {
 #[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize)]
 pub struct PosColorNorm {
     /// Position of the vertex in 3D space.
-    pub position: [f32; 3],
+    pub position: Point3<f32>,
     /// RGBA color value of the vertex.
     pub color: [f32; 4],
     /// Normal vector of the vertex.
-    pub normal: [f32; 3],
+    pub normal: Vector3<f32>,
```

Comment:
yay finally!

Reply:
Oh yeah I started on this didn't I... Oops, guess I might as well finish it.